hack/test-integration.sh

```diff
@@ -13,14 +13,48 @@ cd "${OS_ROOT}"
 
 os::build::setup_env
 
-function cleanup()
-{
-  [ ! -z "${ETCD_STARTED-}" ] && return
-  set +e
-  kill ${ETCD_PID} 1>&2 2>/dev/null
-  rm -rf ${ETCD_DIR} 1>&2 2>/dev/null
-  echo
-  echo "Complete"
+function cleanup() {
+	set +e
+	kill ${ETCD_PID} 1>&2 2>/dev/null
```

Comment:
> at least check for non-empty $ETCD_PID?

Old code didn't, and it never set `ETCD_STARTED`.  I'm not going to argue that we shouldn't fix these things moving forward, but how much effort needs to go into a cleanup method?  If the kill the fails, so what?